src/vector/layer.rs

```diff
@@ -284,7 +284,7 @@ pub trait LayerAccess: Sized {
     /// Get the name of this layer.
     fn name(&self) -> String {
         let rv = unsafe { gdal_sys::OGR_L_GetName(self.c_layer()) };
```

Comment:
We don't need to free `rv` here because it's owned by the layer?

Reply:
> the layer name (must not been freed)